README.md

```diff
@@ -12,15 +12,15 @@ Here is video of a race (running code from students):
 alt="ROSE Race Car Game" width="860" height="720" border="10" /></a>
 
 In this game, two race cars compete to achieve the most points.
-The race car have to recognize the race track, the obstacles, and the bonus areas;
-calculate the best path to take to avoid the pitfalls; and collect bonus points.
+The race cars have to recognize the race track, the obstacles, and the bonus areas
```

Comment:
Every student program single race car, so I don't think it helps to talk about "cars" here.